lib/oban.ex

```diff
@@ -505,4 +541,18 @@ defmodule Oban do
   end
 
   defp child_name(name, child), do: Module.concat(name, child)
+
+  defp send_signal(name, payload) do
+    name
+    |> child_name("Midwife")
+    |> send({:notification, :signal, json_encode_decode(payload)})
+
+    :ok
+  end
+
+  defp json_encode_decode(map) do
```

Comment:
Introduced this helper function to share the same payload (map with atom keys) between the default mode (broadcasting the start/stop queue event to the connected nodes) and the local mode, the latter expects a map with string keys.

Reply:
We can pass a map with string keys to both and avoid this entirely, i.e.:

```elixir
%{"action" => "stop", "queue" => to_string(queue)}
```